src/lib/constants.js

```diff
@@ -57,6 +74,12 @@ export const API_ERROR = {
   NOT_FILE_UPLINK: 'file doesn\'t exist on uplink',
```

Comment:
We should ignore the short forms. It should be `file does not exist on uplink`

Reply:
the problem is some plugins still use them. I'll remove that when I manage to normalize such plugins.